tests/e2e-openshift/red-metrics/check-metrics.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Could you move this shellscript to a container? This will give us more deterministic results (e.g. curl might not be installed on every test host, may have a different version, etc.)

Something like here: https://github.com/grafana/tempo-operator/blob/3b402384c9e78b4cd0ae075548f4c6cebab82505/tests/e2e/smoketest-with-jaeger/03-verify-traces.yaml

Reply:
+1 I will change it